src/Analyzers/Analyzers/test/MinimalStartupTest.cs

```diff
@@ -53,6 +53,9 @@ public async Task StartupAnalyzer_WorksWithNonImplicitMain()
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
+
+namespace MyApp;
```

Comment:
FYI @BrennanConroy

Reply:
?

Reply:
It's using a file scoped namespace. https://github.com/dotnet/aspnetcore/issues/36552 called out that the logging source generator had trouble with it, and I was doing a little bit of sanity testing.